include/enrico/coupled_driver.h

```diff
@@ -102,6 +105,10 @@ class CoupledDriver {
   //! relaxation aplied to the heat source if not set
   double alpha_T_{alpha_};
 
+  //! Constant relaxation factor for the density, defaults to the
+  //! relaxation applied to the heat source if not set
+  double alpha_d_{alpha_};
```

Comment:
I think this would make it a little more obvious
```suggestion
  double alpha_rho_{alpha_};
```